src/pages/documentation/release-notes/index.md

```diff
@@ -50,26 +54,115 @@ Note that all these extensions must be updated together. Flutter and React plugi
 
+### Android Campaign Standard 2.0.0
+
+* Major version update for [Adobe Campaign Standard workflows](./index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is now available as an [open source project on GitHub](https://github.com/adobe/aepsdk-campaign-android).
+
 ## January 31, 2023
 
+### Android Assurance 2.0.0
+
+* Major version update for [Adobe Experience Platform Assurance](./index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is now available as an [open source project on GitHub](https://github.com/adobe/aepsdk-assurance-android).
+
 ### Android Campaign Classic 2.0.0
 
-* Major version update for [Adobe Campaign Classic workflows](../adobe-campaign-classic/index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This extension library is available as an [open source project on GitHub](https://github.com/adobe/aepsdk-campaignclassic-android/).
+* Major version update for [Adobe Campaign Classic workflows](./index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is now available as an [open source project on GitHub](https://github.com/adobe/aepsdk-campaignclassic-android).
+
+### Android Consent 2.0.0
+
+* Major version update for [Consent for Edge Network](./index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is already available as an [open source project on GitHub](https://github.com/adobe/aepsdk-edgeconsent-android).
+
+### Android Edge 2.0.0
+
+* Major version update for [Edge Network](./index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is now available as an [open source project on GitHub](https://github.com/adobe/aepsdk-edge-android).
+
+### Android EdgeIdentity 2.0.0
+
+* Major version update for [Identity for Edge Network](./index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is already available as an [open source project on GitHub](https://github.com/adobe/aepsdk-edgeidentity-android).
+
+### Android UserProfile 2.0.0
+
+- Major version update for [User Profile](./index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is now available as an [open source project on GitHub](https://github.com/adobe/aepsdk-userprofile-android).
+- The following APIs have been deprecated and will be removed in a future release:
+
+  | Deprecated API                      | Recommended Alternative              |
+  | ----------------------------------- | ------------------------------------ |
+  | `UserProfile.registerExtension()`   | `MobileCore.registerExtensions()`    |
```

Comment:
Nit: This applies to all the extensions and it's covered in migration guide. Do we still need i there?

Reply:
I have removed this from release notes, as we have covered it in other places.